website/pages/api-docs/auth/kubernetes/index.mdx

```diff
@@ -41,7 +41,17 @@ access the Kubernetes API.
 - `issuer` `(string: "")` - Optional JWT issuer. If no issuer is specified, then this plugin will
   use `kubernetes.io/serviceaccount` as the default issuer.
 - `disable_iss_validation` `(bool: false)` - Disable JWT issuer validation. Allows to skip ISS validation.
+- `disable_local_ca_jwt` `(bool: false)` - Disable defaulting to the local CA cert and service account JWT when running in a Kubernetes pod.
 
+### Caveats
+
+Either `kubernetes_ca_cert` or `pem_keys` must be set.
```

Comment:
This is dependent on the value from `disable_local_ca_jwt`?

Reply:
Good question, tried to clarify in 6588f33 and e6d098f.